compiler/pipes/filter-only-actually-used.cpp

```diff
@@ -171,7 +171,8 @@ IdMap<FunctionPtr> calc_actually_used_having_call_edges(std::vector<FunctionAndE
       fun->is_main_function() ||
       fun->type == FunctionData::func_class_holder || // classes should be carried along the pipeline
       (fun->is_extern() && vk::any_of_equal(fun->name, "wait", "make_clone")) ||
-      fun->kphp_lib_export;
+      fun->kphp_lib_export ||
+      vk::ends_with(fun->name, ClassData::NAME_OF_TO_STRING);
```

Comment:
Can we use assumptions instead of type inferring? If we use the assumptions we can move conv replacements before this pipe and filter unused `__toString()` methods. Have you thought about it?

Reply:
Since it was decided during the discussion that it was 
too costly for this task, we leave the version as it is.